### ISSUE TITLE
Allow users to exclude specific Sail UI components in their `tailwind.config.js` files

### ISSUE THREAD

**Issue by matt-daneshvar** [sailui/ui#12]
Following Tailwind's docs on [exposing options](https://tailwindcss.com/docs/plugins#exposing-options), Sail UI should allow users to specify a `theme.sailui.exclude` key that instructs the plugin to not include specific components. 

For example, a user who prefers to use the official [tailwindcss-custom-forms](https://github.com/tailwindlabs/tailwindcss-custom-forms) plugin directly, instead of using our form components, should be able to write: 
```js
module.exports = {
  //..
  theme: {
    sailui: {
      exclude: ['forms']
    },
  },,
  plugins: [
    require('sailui'),
  ],
}
```

The API for this feature needs more research.

**Comment by fakeheal** [comment]
I created a PR attempting to implement this functionality: https://github.com/sailui/ui/pull/19

**Comment by matt-daneshvar** [comment]
I just merged #19 and also bumped up the package version to `v0.7.0` to include this. Kudos for the flawless PR and thanks again!
